ansible/roles/bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Ensures bootkube dir exists
+  file: path=/tmp/bootkube state=directory
+
+- name: Extract bootkube binaries
+  unarchive:
+    src: https://github.com/kubernetes-incubator/bootkube/releases/download/v0.3.9/bootkube.tar.gz
+    dest: /tmp/bootkube
+    remote_src: True
+
+- name: Render bootkube manifests
+  command: /tmp/bootkube/bin/linux/bootkube render --asset-dir=/tmp/bootkube/assets --experimental-self-hosted-etcd --etcd-servers=http://10.3.0.15:2379 --api-servers=https://{{ inventory_hostname }}:443
```

Comment:
I should of recognized the subnet and realized this was an incluster-address.  Thanks.

Can we assume the work occurring in kubeadm, i.e. `discovery-file` and `discovery-url` (perhaps in a different form) will be available in the underlying `bootkube` -- Trying to determine if to leverage this we will neet to pivot from raw bootkube to kubeadm at some point or we can assume all work occurring in kubeadm will be available at the lower levels as well.  I don't know how tightly coupled the two are.